### DISCUSSION TITLE
Is this suitable with Mercedes cars?

### DISCUSSION THREAD

**Discussion by tabekg** [cedricp/ddt4all#1032] [Q&A] [answered]
Can I change the configurations of my Mercedes w211 car using this software?

**Comment by Furtif** [comment]
normally not

**Reply by tabekg** [reply]
If I buy a Mercedes database from [here](https://www.ecufiles.com/original-ecu-file/mercedes-benz-e320-3-2-v6-224hp-164-8kw-2005-original-ecu-file/) . Anyway, does not this work?

**Reply by Furtif** [reply, marked as answer]
This is not made for Mercedes

**Reply by Furtif** [reply]
and possibly this database is incompatible, and it won't work with Dtt4all
